private.key

```diff
@@ -1,4 +1,4 @@
 -----BEGIN EC PRIVATE KEY-----
-MDECAQYEICIOHaB1f/ItyzJ66hcLFQmEypsV6VN4HHp2u28/HzCToAoGCCqGSM49
+MDECAQYEIFbjlZzOCsEvX2mzAwaxAVhHEFUUpsGNLtSKA7+XybdHoAoGCCqGSM49
```

Comment:
Why did this key change?

Reply:
I generated a service key and am using that here so that I could get the service key ID and use it on the config file and JSON signing. I know the `kid` is an optional field in the JWT, but I wanted to add it since it follows the pattern we are recommending to users with the new service key change.